src/progressbar/__tests__/progressbar.test.js

```diff
@@ -0,0 +1,60 @@
+/*
+Copyright (c) 2018 Uber Technologies, Inc.
+
+This source code is licensed under the MIT license found in the
+LICENSE file in the root directory of this source tree.
+*/
+// @flow
+import React from 'react';
+import {mount} from 'enzyme';
+import {ProgressBar, StyledLabel} from '../index';
+
+describe('Stateless progress bar', function() {
+  let wrapper,
+    handlers = {};
+  let allProps: any = {},
+    mockFn;
+
+  beforeEach(function() {
+    mockFn = jest.fn();
+    handlers = {
+      getProgressLabel: mockFn,
+    };
+    allProps = {
+      value: 75,
+      ...handlers,
+    };
+  });
+
+  afterEach(function() {
+    jest.restoreAllMocks();
+    wrapper && wrapper.unmount();
+  });
+
+  test('should render component', function() {
+    wrapper = mount(<ProgressBar {...allProps} />);
+    expect(wrapper).toMatchSnapshot('Component has correct render');
+  });
+
+  test('should not render label', function() {
+    allProps.showLabel = false;
+    wrapper = mount(<ProgressBar {...allProps} />);
+    const label = wrapper.find(StyledLabel);
+    expect(label.length).toBe(0);
```

Comment:
Could you add a test where we check that the label is actually rendered when the `showLabel=true`?

Reply:
this is included in default render test. If someone changes it, that test is to be red.

Reply:
I think a snapshot with `MockStyledComponent`s in it is not as useful for testing if the `StyledLabel` is rendered, and is not as readable or easy to understand what's the issue if the test fails

Reply:
I am not a fan of snapshots as well, but when we discussed, why should we use `jest` I was told about snapshots as powerful tool absent in jasmine or other similar frameworks. I should agree that it's handy despite it may not be readable for those, who are not so familiar with it. So if we would like to go back from snapshots here, we should have a test for every single default property for all components. This way we will make it readable as it should be.

Reply:
I did not say that I am not a fan of it. What I'm saying is there are different use cases where a snapshot is not the best approach for checking if a specific component rendered.